core/server/src/main/java/alluxio/worker/netty/DataServerHandler.java

```diff
@@ -68,11 +50,9 @@
    * @param configuration Alluxio configuration
    */
   public DataServerHandler(final BlockWorker blockWorker, Configuration configuration) {
```

Comment:
Should DataServerHandler() just take in an already constructed BlockDataServerHandler(), so it just has to hold the object, and delegate the correct messages? It doesn't seem like DataServerHandler needs to care about BlockWorker, or Configuration.